zerver/tests/tests.py

```diff
@@ -665,6 +665,53 @@ def test_add_bot_with_default_sending_stream_not_subscribed(self):
         profile = get_user_profile_by_email('hambot-bot@zulip.com')
         self.assertEqual(profile.default_sending_stream.name, 'Rome')
 
+    def test_bot_add_subscription(self):
```

Comment:
Similar comment here about the commit message.  Say something like `Add test_bot_add_subscriptions().` or `Test adding subscriptions to bots.`